room_server/test/room_server_test.exs

```diff
@@ -0,0 +1,130 @@
+defmodule RoomServerTest do
+  use ExUnit.Case
+  doctest RoomServer
+
```

Comment:
This constraint kinda breaks the idea of having automated tests :)

Also, it doesn't seem necessary—they all pass if run together.